supply-chain-security/compromises/compromise-definitions.md

```diff
@@ -0,0 +1,39 @@
+# Types of Supply Chain Compromise
+
```

Comment:
I think a list with anchor links to the different definitions in the doc would be nice

Reply:
i suck at markdown but okay.